tests/client/core/api/test_api.js

```diff
@@ -142,6 +142,36 @@ describe('api', () => {
     });
   });
 
+  describe('featured add-ons api', () => {
+    function mockResponse() {
+      return Promise.resolve({
+        ok: true,
+        json() {
+          return Promise.resolve({
+            results: [
+              { slug: 'foo' },
+              { slug: 'food' },
+              { slug: 'football' },
+            ],
+          });
+        },
+      });
+    }
+
+    it('sets the app, lang, and type query', () => {
+      // FIXME: This shouldn't fail if the args are in a different order.
```

Comment:
I think the order of `Object.keys({ clientApp, lang })` is guaranteed since ES6 so it should always work.